tock/hours/models.py

```diff
@@ -10,11 +10,13 @@
 
 
 class ReportingPeriod(ValidateOnSaveMixin, models.Model):
+    """Reporting period model details"""
     start_date = models.DateField(unique=True)
-    end_date = models.DateField()
-    working_hours = models.PositiveSmallIntegerField(
-        default=40,
-        validators=[MaxValueValidator(80)])
+    end_date = models.DateField(unique=True)
+    exact_working_hours = models.PositiveSmallIntegerField(
```

Comment:
Is there a corresponding data migration with this change?  Meaning, for all records we have that were stored in the `working_hours` property, is there a migration to shift that data over into this new `exact_working_hours` field?  I'm not seeing one above so I wanted to double check.  If there's not, we need to add one so we do not lose existing data in the system.